src/sentry/api/bases/project.py

```diff
@@ -111,10 +111,18 @@ class ProjectEndpoint(Endpoint):
     def convert_args(
         self,
         request: Request,
-        organization_slug: str | int,
         *args,
         **kwargs,
     ):
+        if args and args[0] is not None:
+            organization_id_or_slug: int | str = args[0]
+            # Required so it behaves like the original convert_args, where organization_id_or_slug was another parameter
+            args = args[1:]
+        else:
+            organization_id_or_slug = kwargs.pop("organization_id_or_slug", None) or kwargs.pop(
+                "organization_slug"
+            )
+
         if args and args[0] is not None:
             project_id_or_slug: int | str = args[0]
```

Comment:
This is a temporary jerry rig so that we maintain backwards compatibility with `getsentry` endpoints until I update them. They expect convert_args to accept `organization_slug`, so I can't keep it named, I have to keep it unnamed and extract it. I don't like it, but couldn't think of a better way.

Reply:
Please add a TODO comment and make sure they will be done right after. I see your point.

Reply:
Nope it isn't, Org is above Project in the hierarchy